README.md

```diff
@@ -33,6 +33,13 @@ See the [docs](http://docs.datahub.io/developers/) for more information.
 
 ## Developers
 
+Storage manager api doesn't allow requests coming from localhost or any other locations.
```

Comment:
This statement is true but is not related to `git submodules`. That error will be printed anyway but wouldn't crash the process - showcase page still will be available with `N/A` message for the size of the dataset.

Can you please remove this sentence and the following one? Instead of that we should say something like:

> The javascript portion of the app, which is responsible for rendering views, comes from a different repo. That repo must be submoduled in and then built: